lib/features/email/domain/usecases/parse_calendar_event_interactor.dart

```diff
@@ -30,21 +30,18 @@ class ParseCalendarEventInteractor {
         eagerError: true
       );
 
-      final listCalendarEvent = List<CalendarEvent>.empty(growable: true);
+      final mapCalendarEvent = <Id, List<CalendarEvent>>{};
```

Comment:
why we need use map?

Reply:
To know which `Id` did the controller choose, to make the `reply` action doable.